packages/ndla-ui/src/RelatedArticles/RelatedArticles.jsx

```diff
@@ -19,7 +19,7 @@ const RelatedArticles = ({ resources }) => (
         <div key={uuid()} {...classes('item')}>
```

Comment:
Jeg tror vi skal utvide denne oppgaven for å gjøre komponenten mer gjenbrukbar.

* Split det i to komponenter RelatedArticleList og RelatedArticle (er greit å ha de i samme fil).
* RelatedArticle er det som er inne i map funksjonen. I RelatedArticleList bruker du `children` prop'en istedenfor `resource.filter`
* Endre mappe- of filnavn fra RelatedArticles -> RelatedArticleList
* Legge til en index.js som eksporterer RelatedArticleList som default og RelatedArticle (se hvordan det er gjort med MastHead)
* `Relaterte artikler` teksten må sendes inn som en prop til RelatedArticleList
* Foreslår at RelatedArticle tar inn tre separate props `title`, `introduction` og `to` (linken). Eventuelt så kan du sende inn en article prop med title og introduction (hvis det er mer naturlig for hvordan den vil blir brukt i article-converter. Tror `to` bør være en separat prop. 
* Fjern path fra mockResources.js etter omskrivingen. \# kan sendes inn direkte.

Tanken er at den nå kan brukes slik:
```js
<RelatedArticleList message={{title: '...'}}>
  <RelatedArticle title={} intro={} to={} />
  <RelatedArticle title={} intro={} to={} />
</RelatedArticleList>

eller

<RelatedArticleList message={{title: '...'}}>
  {articles.map(article => (<RelatedArticle article={} to={} />))}
</RelatedArticleList>
```